Task1/script.js

```diff
@@ -0,0 +1,42 @@
+/*let randn_bm = function() {  
+    let u = 0, v = 0;  
+        while (u === 0) 
+            u = Math.random();  
+        while (v === 0) 
+            v = Math.random();  
+    return Math.sqrt(-2.0 * Math.log(u)) * Math.cos(2.0 * Math.PI * v) * 2 | 0;
+};*/
+
+const array = [];
+const result = {};
+
+let randn_bm = function(){
+    while (array.length < 10) {
+        let u = 0;
+        let v = 0;  
+            while (u === 0) {
+                u = Math.random(); 
+            }
+            while (v === 0) {
+                v = Math.random(); 
+            }
+
+    let number = Math.sqrt(-2.0 * Math.log(u)) * Math.cos(2.0 * Math.PI * v) * 2 | 0;
+    array.push(number);
```

Comment:
this function should return a random number and you need to update the array outside of this function